packages/customize-widgets/src/components/sidebar-block-editor/sidebar-adapter.js

```diff
@@ -53,12 +48,25 @@ export default class SidebarAdapter {
 		this.setting = setting;
 		this.allSettings = allSettings;
 
+		this.locked = false;
+		this.widgetsMap = new WeakMap();
 		this.subscribers = new Set();
 
+		this.history = [
+			this._getWidgetIds().map( ( widgetId ) =>
+				this.getWidget( widgetId )
+			),
+		];
+		this.historyIndex = 0;
```

Comment:
These properties could be defined at the class level if you wanted to clean up the constructor a little. Up to you.

```js
class SidebarAdapter {
	locked = false;

	widgetsMap = new WeakMap();

	subscribers = new Set();

	history = [];

	historyIndex = 0;

	constructor( setting, allSettings ) {
		this.setting = setting;
		this.allSettings = allSettings;

		this.history = [
 			this._getWidgetIds().map( ( widgetId ) =>
 				this.getWidget( widgetId )
 			),
 		];

		this._handleSettingChange = this._handleSettingChange.bind( this );
 		this._handleAllSettingsChange = this._handleAllSettingsChange.bind( this );
	}
}
```

Reply:
It's a syntax error though since that we never enabled class properties in our babel settings 😅.